library/src/scripts/components/forms/ErrorMessages.tsx

```diff
@@ -1,11 +1,12 @@
-/**
+/*
+ * @author Stéphane LaFlèche <stephane.l@vanillaforums.com>
  * @copyright 2009-2018 Vanilla Forums Inc.
  * @license GPL-2.0-only
  */
 
 import React from "react";
 import classNames from "classnames";
-import { getRequiredID, IRequiredComponentID } from "@library/componentIDs";
+import { getRequiredID, IRequiredComponentID } from "../../componentIDs";
```

Comment:
I am not sure if that is best practice to have relative paths.
We can lose it during build and deploy phases

Reply:
Good catch!